draft-ietf-httpbis-digest-headers.md

```diff
@@ -213,57 +215,42 @@ interpreted as described in {{SEMANTICS}}.
 Algorithm names respect the casing used in their definition document (e.g. SHA-1, CRC32c)
 whereas digest-algorithm tokens are quoted (e.g. "sha", "crc32c").
 
-# Representation Digest {#representation-digest}
-
-The representation digest is an integrity mechanism for HTTP resources
-which uses a checksum  that is calculated independently of the content
-(see {{Section 6.4 of SEMANTICS}}).
-It uses the representation data (see {{Section 8.1 of SEMANTICS}}),
-that can be fully or partially contained in the content, or not contained at all.
+# The Digest Field {#digest}
 
-This takes into account the effect of the HTTP semantics on the messages;
-for example, the content can be affected by Range Requests or methods such as HEAD,
-while the way the content is transferred "on the wire" is dependent on other
-transformations (e.g. transfer codings for HTTP/1.1 - see {{Section 6.1 of
-HTTP11}}). To help illustrate how such things affect `Digest`,
-several examples are provided in {{resource-representation}}.
+The `Digest` field contains a comma-separated list of one or more representation
+digest values. It can be used in both requests and responses.
 
-A representation digest consists of
-the value of a checksum computed on the entire selected `representation data`
-(see {{Section 8.1 of SEMANTICS}}) of a resource identified according to {{Section 6.4.2 of SEMANTICS}}
-together with an indication of the algorithm used:
+A representation digest is an integrity mechanism for HTTP resources that uses a
```

Comment:
I liked having a separate sub/section for that... let me think

Reply:
By merging the section we end up with 9 total paragraphs in one section vs 12 total paragraphs split across 2 sections. That seems editorial cleaner to me.

It also makes the section closer to the layout of Content Digest, which seems beneficial when reading both.

But if you have an alternative proposal I'm happy to review it.